nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -121,26 +121,48 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
         else:
             augmentor = None
 
-        dataset = AudioToBPEDataset(
-            manifest_filepath=config['manifest_filepath'],
-            tokenizer=self.tokenizer,
-            sample_rate=config['sample_rate'],
-            int_values=config.get('int_values', False),
-            augmentor=augmentor,
-            max_duration=config.get('max_duration', None),
-            min_duration=config.get('min_duration', None),
-            max_utts=config.get('max_utts', 0),
-            trim=config.get('trim_silence', True),
-            load_audio=config.get('load_audio', True),
-            add_misc=config.get('add_misc', False),
-        )
+        shuffle = config['shuffle']
+
+        # Instantiate tarred dataset loader or normal dataset loader
+        if 'is_tarred' in config and config.get('is_tarred', False):
+            dataset = TarredAudioToBPEDataset(
+                audio_tar_filepaths=config['tarred_audio_filepaths'],
+                manifest_filepath=config['manifest_filepath'],
+                tokenizer=self.tokenizer,
+                sample_rate=config['sample_rate'],
+                int_values=config.get('int_values', False),
+                augmentor=augmentor,
+                shuffle_n=config.get('shuffle_n', 50),
```

Comment:
Just realized I left the arbitrarily set `shuffle_n=50` in `ctc_models.py`... Do you mind adding this as well that file as well?
And I'm not sure that 50 is a good default (sorry about that); maybe something like 64 or 128, some multiple of the likely batch size.

Reply:
Yup sounds good.

Reply:
Done, im defaulting to 4x batch size as a reasonable buffer for shuffle